README.md

```diff
@@ -4,31 +4,34 @@ A starter boiler plate for node application with authentication, which includes
 
 ## Getting Started
 
-1. ### Installation
+1. ### Clone the repo using
+```git clone https://github.com/soha-moosa/node-auth-kit.git```
+
+2. ### Installation
 
 - Run `npm install` to install all project dependencies available in package.json file.
 
-2. ### Add nodemon.json
+3. ### Add nodemon.json
 
-- add nodemon.json file with your data
+- Add nodemon.json file with your data at your root directory.
 
 ```{
   "env": {
-    "Node_ENV": "",
-    "PORT": ,
-    "SECRET": "",
-    "MONGODB_URI": "",
-    "SEND_GRID_API_KEY": "",
-    "JWT_SUBJECT": "",
-    "FACEBOOK_APP_ID": "",
-    "FACEBOOK_APP_SECRET": "",
-    "GOOGLE_CLIENT_ID": "",
-    "GOOGLE_CLIENT_SECRET": ""
+    "Node_ENV": "", // Add your development environment mode
+    "PORT": , // Server will listen at this PORT
+    "SECRET": "", // Add SECRET to be used in sessions or jwt
+    "MONGODB_URI": "", // Add your mongodb atlast connection URL here
```

Comment:
Typo: atlast 
Plus this can be used with the local database. So please change this to : Your mongodb database connection  URL.